kong/tests/test_kong.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+import os
+import subprocess
+import logging
+import requests
+import time
+
+from datadog_checks.kong import Kong
+from datadog_checks.checks import AgentCheck
```

Comment:
Can we remove this and just do e.g. `Kong.OK`.

Reply:
There's no different but sure